HTML tag changes break tests

```
org.junit.ComparisonFailure: expected:<"Kinder erfreuen.<br[ /><br /]>Make children happy...> but was:<"Kinder erfreuen.<br[><br]>Make children happy...>
```

As the trackable seems orphaned, I assume either a small website change/glitch.
Or did we have any related changes causing this html tag changing?

there is also a second error:
~~~
testSearchByGeocodeBasis(cgeo.geocaching.CgeoApplicationTest)
java.lang.AssertionError: [description]
Expecting:
<"<img src="http://img.geocaching.com/cache/large/1711f8a1-796a-405b-82ba-8685f2e9f024.jpg">">
to start with:
<"<img src="http://img.geocaching.com/cache/large/1711f8a1-796a-405b-82ba-8685f2e9f024.jpg" />">
~~~

--- a/tests/src-android/cgeo/geocaching/CgeoApplicationTest.java
+++ b/tests/src-android/cgeo/geocaching/CgeoApplicationTest.java
@@ -78,7 +78,7 @@ public class CgeoApplicationTest extends CGeoTestCase {
         assertThat(tb.getOrigin()).isEqualTo("Niedersachsen, Germany");
         assertThat(tb.getOwner()).isEqualTo("blafoo");
         assertThat(tb.getOwnerGuid()).isEqualTo("0564a940-8311-40ee-8e76-7e91b2cf6284");
-        assertThat(tb.getGoal()).isEqualTo("Kinder erfreuen.<br[><br]>Make children happy.");
+        assertThat(tb.getGoal()).isEqualTo("Kinder erfreuen.<br><br>Make children happy.");
         assertThat(tb.getDetails()).startsWith("Auf der CD sind");
         // the host of the image can vary
         assertThat(tb.getImage()).endsWith("geocaching.com/track/large/38382780-87a7-4393-8393-78841678ee8c.jpg");

--- a/tests/src-android/cgeo/geocaching/CgeoApplicationTest.java
+++ b/tests/src-android/cgeo/geocaching/CgeoApplicationTest.java
@@ -78,7 +78,7 @@ public class CgeoApplicationTest extends CGeoTestCase {
         assertThat(tb.getOrigin()).isEqualTo("Niedersachsen, Germany");
         assertThat(tb.getOwner()).isEqualTo("blafoo");
         assertThat(tb.getOwnerGuid()).isEqualTo("0564a940-8311-40ee-8e76-7e91b2cf6284");
-        assertThat(tb.getGoal()).isEqualTo("Kinder erfreuen.<br /><br />Make children happy.");
+        assertThat(tb.getGoal()).isEqualTo("Kinder erfreuen.<br[><br]>Make children happy.");
         assertThat(tb.getDetails()).startsWith("Auf der CD sind");
         // the host of the image can vary
         assertThat(tb.getImage()).endsWith("geocaching.com/track/large/38382780-87a7-4393-8393-78841678ee8c.jpg");

--- a/tests/src-android/cgeo/geocaching/test/mock/GC2JVEH.java
+++ b/tests/src-android/cgeo/geocaching/test/mock/GC2JVEH.java
@@ -83,7 +83,7 @@ public class GC2JVEH extends MockedCache {
 
     @Override
     public String getDescription() {
-        return "<img src=\"http://img.geocaching.com/cache/large/1711f8a1-796a-405b-82ba-8685f2e9f024.jpg\" />";
+        return "<img src=\"http://img.geocaching.com/cache/large/1711f8a1-796a-405b-82ba-8685f2e9f024.jpg\">";
     }
 
     @Override

--- a/tests/src/cgeo/geocaching/test/mock/GC2JVEH.html
+++ b/tests/src/cgeo/geocaching/test/mock/GC2JVEH.html
@@ -601,7 +601,7 @@ Sanne</p>
         <br />
         <div class="UserSuppliedContent">
             
-            <span id="ctl00_ContentBody_LongDescription"><img src="http://img.geocaching.com/cache/large/1711f8a1-796a-405b-82ba-8685f2e9f024.jpg" /></span>
+            <span id="ctl00_ContentBody_LongDescription"><img src="http://img.geocaching.com/cache/large/1711f8a1-796a-405b-82ba-8685f2e9f024.jpg"></span>
             
         </div>
         

